charms/istio-pilot/tests/conftest.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+from charm import Operator
+from ops.testing import Harness
+
+
+class Helpers:
+    @staticmethod
+    def get_unique_calls(call_args_list):
```

Comment:
please see my above comments about hashable items and sets and how to `mock_obj.assert_*` methods